packages/ra-core/src/form/getSimpleValidationResolver.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { getSimpleValidationResolver } from './getSimpleValidationResolver';
+
+describe('getSimpleValidationResolver', () => {
+    const validator = getSimpleValidationResolver(errors => errors);
+
+    it('should recieve a flattened errors object', async () => {
+        const errors = await validator({
```

Comment:
that's really confusing: validator should receive an object of values, and here the tests makes it look like it expects errors... Could you find a way to test it that it clearer? Perhaps by unit testing the flattenKeys function